airflow/version.py

```diff
@@ -17,4 +17,13 @@
 # under the License.
 #
 
-version = '2.0.0b3'
+__all__ = ['version']
+
+try:
+    from importlib import metadata
+except ImportError:
+    import importlib_metadata as metadata
+
+version = metadata.version('apache-airflow')
+
+del metadata
```

Comment:
(This line isn't strictly needed, it was just so that the only symbol in this module was `version`. For no god reason really)